modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,326 @@
+= MongoDB - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Upgrade Anypoint Connector for MongoDB (MongoDB Connector) to version 6.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|5.3.x |6.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Changed connection types and configuration
+* Changed operation names, metadata, fields, and values
+* Changed input sources
+
+=== Changes in Connection Types and Configuration
+
+This release contains the following changes to connection types and configuration.
+
+* Removed the Connection String connection type. There is now a single, unified generic connection that contains the configurable properties that were previously available in the Connection String connection type.
+* Removed the Metadata configuration section. MongoDB 6.0 automatically generates the metadata.
+
+==== Config Advanced Tab
+
+The following optional changes apply to the Advanced tab in the global configuration:
+
+* Added the following read concern options:
+** Read concern: Enables the control of the consistency and isolation properties of the data read from replica sets and replica set shards.
+** Read preference: Specifies the read preferences for this connection.
+** Maximum staleness seconds: Specifies how stale a secondary can be before the client stops using it for read operations. A value of -1 means that there is no maximum staleness threshold.
+** Read preference tags: Enables you to specify a tag set in the read preference to target those members associated with a tag in the set.
+* Added the following write concern options, with the following fields:
+** Write concern acknowledgment: Requests an  acknowledgment that the write operation was propagated to the specified number of instances.
+** Write concern timeout: Specifies a time limit for the write concern.
+** Write concern timeout timeunit: Specifies a time unit for the Write concern timeout field.
```

Comment:
Is the field named "Write concern timeout"?
If so...I fear we might have to go to bold for these labels, to make them distinct from the surrounding text. But hold off: Mysti is putting together a bold and mono proposal because our use is all over the place....